src/types/json.h

```diff
@@ -198,6 +198,23 @@ struct JsonValue {
     return count;
   }
 
+  Status ArrLen(std::string_view path, std::vector<std::optional<uint64_t>> &arr_lens) const {
+    try {
+      jsoncons::jsonpath::json_query(value, path,
+                                     [&arr_lens](const std::string & /*path*/, const jsoncons::json &basic_json) {
+                                       if (basic_json.is_array()) {
+                                         arr_lens.emplace_back(static_cast<uint64_t>(basic_json.size()));
+                                       } else {
+                                         arr_lens.emplace_back();
```

Comment:
This code works, however, I think `arr_lens.emplace_back(std::nullopt);` is better for understanding here